lib/matplotlib/font_manager.py

```diff
@@ -1117,12 +1118,11 @@ def score_family(self, families, family2):
                 options = [x.lower() for x in options]
                 if family2 in options:
                     idx = options.index(family2)
-                    return ((0.1 * (idx / len(options))) *
-                            ((i + 1) / len(families)))
+                    return (i + (idx / len(options))) * step
```

Comment:
A match for an alias should never be higher than an exact match.  Yet here, if i == 0, the return value is 0.